Bendy and the Dark Revival/LiveSplit.BendyAndTheDarkRevival.asl

```diff
@@ -15,8 +17,9 @@ startup
         the idea is that the key here is the setting that's actually checked in-code
      */
     vars.SettingAliases = new Dictionary<string, List<string>>() {
```

Comment:
Can't this just be a `Dictionary<string, string>`?

Reply:
um, sure it could, but I had reason to believe during implementation that a 1 to many mapping would be necessary

Reply:
Feel free to private the repository.

Reply:
Ero, I shouldn't need to say this, but not only is your comment plain unhelpful, chiming in for no reason on a PR for an unrelated section of code is just not rude.

I don't want to deviate into ways I could stop you from doing this - obviously I could private the repo and everything I've ever made, breaking all my autosplitters until someone updates the links with a fork - but this is a straw man. I'm not saying you don't have the right to comment on this, I'm letting you know that they are not appreciated and therefore will be ignored in the future.

At the least, you need to keep these comments on-topic to the PR. Raise a PR or an issue if you think an issue is worth raising.